docs/components/radio-field/demo/base-demo.md

```diff
@@ -9,6 +9,7 @@
   />
   <Form::Fields::Radio
     @label='Option 2'
+    @hint='Some hint'
```

Comment:
nit: When I wrote this initially, I didn't add a `@hint` here on purpose so that both uses could be shown (with label + with label + hint).  Would it be okay to axe this change?